crypto/keys/lazy_keybase.go

```diff
@@ -221,4 +210,26 @@ func (lkb lazyKeybase) ExportPrivKey(name string, decryptPassphrase string,
 	return newDBKeybase(db, lkb.options...).ExportPrivKey(name, decryptPassphrase, encryptPassphrase)
 }
 
+// SupportedAlgos returns a list of supported signing algorithms.
+func (lkb lazyKeybase) SupportedAlgos() []SigningAlgo {
+	db, err := sdk.NewLevelDB(lkb.name, lkb.dir)
```

Comment:
Why do we open a DB and then close it? Can we call `SupportedAlgos` w/o needing to open a DB?